### PR TITLE
added oak9 and torq to the list

### PR DESCRIPTION
**Pull request by gautambaghel**
### What
Added Torq and Oak9 to the Run Tasks partner list

### Why
They've finished their Run Tasks integration

### Screenshots
N/A

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [ ] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [ ] Description links to related pull requests or issues, if any.

#### Content
- [ ] Redirects have been added for moved, renamed, or deleted pages. This requires a separate PR in the [`terraform-website` repository](https://github.com/hashicorp/terraform-website) `redirects.next.js` file.
- [ ] API documentation and the API Changelog have been updated. 
- [ ] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [ ] Pages with related content are updated and link to this content when appropriate.
- [ ] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [ ] New pages have metadata (page name and description) at the top.
- [ ] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [ ] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [ ] UI elements (button names, page names, etc.) are bolded.
- [ ] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

**Comment by gautambaghel** [comment]
@laurapacilio -- Waiting for your review!

**Comment by laurapacilio** [comment]
hooray! I'm going to tag in our new technical writer @mgarrell777 on this. Matthew - thank you very much for your help here! @gautambaghel you are in good hands :)

File: website/docs/cloud-docs/integrations/run-tasks/index.mdx [hunk 1]
```diff
@@ -139,6 +139,12 @@ To get started, [sign up](https://www.lightlytics.com/signup) for a free trial,
 
 To get started, request early access at [Moderne.io](https://moderne.io/), and follow the instructions in the [Terraform Cloud integration](https://docs.moderne.io/how-to/terraform-cloud-integration) user documentation.
 
+### Oak9
+
+[Oak9](https://oak9.io/) builds cloud native security into application design so you can develop fast and innovate. Dynamically secure Infrastructure as Code (IaC) and deployed cloud native workloads.
```
Comment (mgarrell777) [inline website/docs/cloud-docs/integrations/run-tasks/index.mdx:144]:
```suggestion
[Oak9](https://oak9.io/) dynamically secures your Infrastructure as Code (IaC) and deployed cloud native workloads using built-in cloud native security for application designs, allowing you to innovate and develop quickly.
```

**Review by mgarrell777 — APPROVED** [review]
Hi @gautambaghel I'm Matthew, new technical writer for Terraform. Thanks very much for your work on this PR. I have just a couple small suggestions for edits. Otherwise, looks good to me.

**Comment by gautambaghel** [comment]
Hey @mgarrell777 ! Great to have you at HashiCorp, awesome suggestions. I've incorporated all of them. Thanks!

**Review by mgarrell777 — APPROVED** [review]
Looks good to me. Thanks again @gautambaghel

**Comment by gautambaghel** [comment]
@mgarrell777 I don't have access to merge this PR, would you be able to do that for me pls, thanks!

**Comment by mgarrell777** [comment]
> @mgarrell777 I don't have access to merge this PR, would you be able to do that for me pls, thanks!

@gautambaghel Done. Thanks very much Gautam.

**Pull request merged**